css/register.css

```diff
@@ -0,0 +1,75 @@
+.register-container {
+    display: flex;
+    flex-direction: column;
+    justify-content: flex-start;
+    align-items: flex-start;
+    margin-left: 10px;
+    margin-right: 10px;
+    padding: 15px;
+
+    background-color: var(--main-color);
+    color: white;
+    border-radius: 0px;
+    max-width: 350px;
+}
+
+.register-container > h1 {
+    align-self: center;
+}
+
+body {
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    align-items: center;
+    height: 100vh;
```

Comment:
There is a known issue with `100vh` on mobile screens. Try archiving the same behaviour without it